setuptools/dist.py

```diff
@@ -406,7 +411,8 @@ class Distribution(_Distribution):
         'long_description_content_type': None,
         'project_urls': dict,
         'provides_extras': ordered_set.OrderedSet,
-        'license_files': ordered_set.OrderedSet,
+        'license_file': None,
```

Comment:
I had kind-of hoped this would go away, especially since the value is deprecated. I tried removing it and got an error, but I think it can be eliminated by being more defensive at the one point where it's queried.

Reply:
I'm happy to remove it if you want. Personally, I would also be open to remove the option `license_file` entirely and return an error. Should be easy enough for the user to fix then.